apps/arm/main.cpp

```diff
@@ -13,6 +13,13 @@
 #include "CANBuffer.h"
 #include "Servo.h"
 
+// maybe these can be defined more formally in some config file?
+#define TURNTABLEACTUATORID   0
+#define SHOULDERACTUATORID    1
+#define ELBOWACTUATORID       2
+#define WRISTLEFTACTUATORID   3
+#define WRISTRIGHTACTUATORID  4
```

Comment:
should there be claw as well?

Reply:
hmm not sure. i did not see and ActuatorController Instance for claw